de.dlr.sc.virsat.model/src-gen/de/dlr/sc/virsat/model/dvlm/categories/ATypeInstance.java

```diff
@@ -70,7 +70,6 @@
 	 * Call this method to get a CategoryAssignmentContainer handed back for any TypeInstance. This method returns null in case it could not be found.
 	 * <!-- end-model-doc -->
 	 * @model kind="operation"
-	 *        annotation="http://www.eclipse.org/emf/2002/GenModel body='return de.dlr.sc.virsat.model.dvlm.categories.util.CategoryAssignmentHelper.getContainerFor(this);'"

```

Comment:
Hey @andriikovalov-dlr , why did all these annotations dissappear?

Reply:
Discussed together with @andriikovalov-dlr . This change could be a result of migrating to a newer eclipse and changes in EMD. Nevertheless, release notes of EMF do not give evidence. Besides that the annotations are now written to the genmodel rather the generated interfaces, nothing has changed.

Reply:
It looks like the new EMF code generator doesn't put these annotations into javadoc (like here), but includes it into .genmodel